src/vs/workbench/contrib/positronVariables/browser/components/variablesInstance.tsx

```diff
@@ -513,6 +513,17 @@ export const VariablesInstance = (props: VariablesInstanceProps) => {
 		}
 	};
 
+	// workaround for web disabling scrolling on the window to prevent URL navigation
+	useEffect(() => {
+		if (!isWeb) {
+			return;
+		}
+
+		outerRef.current?.addEventListener('wheel', (e: WheelEvent) => {
+			innerRef.current.parentElement?.scrollBy(e.deltaX, e.deltaY);
+		});
```

Comment:
Is there a reasonable way to pull this `onWheel` logic into a util function that can be reused across all the places we are adding this handling? It could make it easier to keep track of the places we have this handling, as well as have a single place to update the implementation if needed.

Reply:
I think the easiest way to search for all instances currently is using the search term `scrollBy(e.deltaX, e.deltaY)`

Reply:
I think creating a custom React hook might be able to fulfill this. As long as it has access to the element that receives the scroll event and the element that needs to receive the scroll data (they can be different). This might be an easier way to deal with this bug over using the `Scrollable` so that it's more React compatible.